token-lending/program/src/processor.rs

```diff
@@ -2227,6 +2228,11 @@ fn get_switchboard_price(
     if *switchboard_feed_info.key == spl_token_lending::NULL_PUBKEY {
         return Err(LendingError::NullOracleConfig.into());
     }
+    if switchboard_feed_info.owner == &spl_token_lending::SWITCHBOARD_V2_MAINNET
```

Comment:
Instead of this check, could add a reserve config field which tracks which version of the switchboard program is used.

Reply:
i think in 4~5 months switchboard v1 will not exist so, will end up removing old code path and putting the program address on the market account for switchbaord v2